Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
     msgpack (1.4.2)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
     net-ssh (6.1.0)
-    nio4r (2.5.5)
-    nokogiri (1.11.1-x86_64-darwin)
+    nio4r (2.5.7)
+    nokogiri (1.11.3-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
-    pagy (3.10.0)
+    pagy (3.11.0)
     parallel (1.20.1)
-    parser (3.0.0.0)
+    parser (3.0.1.0)
```

Comment:
### [parser](https://github.com/whitequark/parser)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.0.0.0 → 3.0.1.0

[_[change-log](https://github.com/whitequark/parser/blob/v3.0.1.0/CHANGELOG.md), [source-code](https://github.com/whitequark/parser/tree/v3.0.1.0)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/whitequark/parser/compare/v3.0.0.0...v3.0.1.0).

These are the individual commits:
- (9d5e851) [Update changelog.](https://github.com/whitequark/parser/commit/9d5e8510d54374be3d992558d81df3735173b09b)
- (b4e903d) [+ `Source::Buffer#inspect` [#728] (#787)](https://github.com/whitequark/parser/commit/b4e903da1f96b0c98af4d74bf6615c9f7516c9ab)
- (de619a2) [+ `Source::TreeRewriter#inspect` [#728] (#788)](https://github.com/whitequark/parser/commit/de619a2ff36a39b4d43efa9b86cdec98621ed435)
- (bbba397) [* Bump maintenance branches to 3.0.1, 2.7.3, 2.6.7, and 2.5.…](https://github.com/whitequark/parser/commit/bbba39722450a808137cd69ddf516199f0dbec6e)
- (287bb8d) [Bump version.](https://github.com/whitequark/parser/commit/287bb8dd44c4d6c5f011a1af9ee895e8c3877c9e)
- (8d8b5dd) [Update changelog.](https://github.com/whitequark/parser/commit/8d8b5dd3287784ea2ed98c1b34277190f25ef7f1)


</details>